tools/provision-alerts/fetch_inventory.sh

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env bash
+
+#create infra agent ansible inventory with ec2 instances matching PATTERN
+PATTERN=$1
+if [ -z "${PATTERN}" ];then
+  echo "pattern cannot be empty"
+  exit 1
+fi
+
+INSTANCES=$( aws ec2 describe-instances --no-paginate \
+  --filters 'Name=instance-state-code,Values=16' \
+  --query 'Reservations[*].Instances[*].{PrivateIpAddress:PrivateIpAddress,Name:[Tags[?Key==`Name`].Value][0][0],InstanceId:InstanceId}' \
+  | jq -c '.[]' \
+  | tr -d "[]{}\""
+
+)
+
```

Comment:
something I thought about but never tried is to add the PATTERN in the describe instances as input so we don't need to do the hack of the ubuntu-runner below :)